source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -308,7 +327,10 @@ namespace AdaptiveCards { namespace XamlCardRenderer
     }
 
     _Use_decl_annotations_
-    ComPtr<IUIElement> XamlBuilder::CreateRootCardElement(IAdaptiveCard* adaptiveCard, IPanel** childElementContainer)
+    ComPtr<IUIElement> XamlBuilder::CreateRootCardElement(
+        IAdaptiveCard* adaptiveCard, 
```

Comment:
>  [](start = 36, length = 1)

[nit] extra space

Reply:
:)

---
In reply to: [140530587](https://github.com/Microsoft/AdaptiveCards/pull/648#discussion_r140530587) [](ancestors = 140530587)